cavern/VERSION

```diff
@@ -0,0 +1,4 @@
+## deployable containers have a semantic and build tag
+# semantic version tag: major.minor
+# build version tag: timestamp
+TAGS="0.2.2 $(date -u +"%Y%m%dT%H%M%S")"
```

Comment:
we eventually found that having separately tagged images didn't help ops at all, so I have been including a dash between the semantic and build, eg 0.2.2-$(...) so there is one TAG with both bits of info

Reply:
I wouldn't want to have to update the k8s deployment info when re-iterating on testing a particular version.  The config refers to, :0.2.2 (for example) and uses the policy to 'Always Pull', meaning it checks for new versions of the ImageID for that tag, but doesn't pull if there are no changes.  I haven't been using the timestamped tag but think it's useful for keeping history.